resources/plugins/skynetiads/mist_4_3_74.lua

```diff
@@ -0,0 +1,6825 @@
+--[[--
```

Comment:
We may want to remove this file as this would be included by default on high up dependencies? E.g. DCS Liberation plugin

Reply:
Good remark, I did it.
Thanks !